app/api/project/[id]/faults/remove/route.ts

```diff
@@ -0,0 +1,26 @@
+import prisma from "@/app/utils/prisma/client";
+import { NextRequest, NextResponse } from "next/server";
+
+export async function DELETE(req: NextRequest) {
+    if (req.method === 'DELETE') {
```

Comment:
We don't need to check the method that's being used, since the function can only be called by sending the DELETE request